common/hooks/pre-pkg/99-pkglint.sh

```diff
@@ -60,6 +60,21 @@ hook() {
 		error=1
 	fi
 
+	# Forbid packages containing only license
+	licensepath="$PKGDESTDIR"
+	for f in usr share licenses; do
+		if [ "$(ls $licensepath)" = "$f" ]; then
+			licensepath+=/$f
+		else
+			licensepath=
+			break
+		fi
+	done
+	if [ "$licensepath" ]; then
```

Comment:
I think we should add `-a "$build_style" != meta`, no?

Reply:
No, the sole purpose of meta "build style" is allowing package to be empty, and once license is included it's not empty anymore. Especially since they are currently marked as BSD, and BSD should be included; but one should not include license file to otherwise empty package.